source/conf.py

```diff
@@ -0,0 +1,164 @@
+# -*- coding: utf-8 -*-
```

Comment:
I would suggest to use the readthedocs theme. Here is an example of how to do that https://github.com/mozilla/python_moztelemetry/blob/master/docs/conf.py#L19-L23